README.md

```diff
@@ -41,3 +41,203 @@ Package can then be placed directly under `./charts/<centos|redhat>/<chart-name>
+Last Completed: Tue Apr  4 10:31:52 2023
+Phase:          Succeeded
+
+POD LOGS: postgresql-persistent-connection-test
+postgresql-testing:5432 - accepting connections
+```
+
+
+In case the Helm chart has already been merged,
+use the following command to verify if all the required information is contained within the Helm chart.
+
+```commandline
+docker run --rm -i -e KUBECONFIG=/.kube/config -v "${HOME}/.kube":/.kube \
+          "quay.io/redhat-certification/chart-verifier:latest" \
+          verify \
+          https://sclorg.github.io/helm-charts/<helm-chart-name>.tgz
+```
+
+Use the following command to verify if all the required information for local testing is contained within the Helm chart.
+
+```commandline
+docker run --rm -i -e KUBECONFIG=/.kube/config -v "${HOME}/.kube":/.kube \
+          -v "<full_path_to_this_dir>":/helm-charts
+          "quay.io/redhat-certification/chart-verifier:latest" \
+          verify \
+          /helm-charts/<helm-chart-name>.tgz
+```
+
+The output from commands mentioned above could be:
+
+```commandline
+results:
+    - check: v1.0/required-annotations-present
+      type: Mandatory
+      outcome: PASS
+      reason: All required annotations present
+    - check: v1.0/signature-is-valid
+      type: Mandatory
+      outcome: SKIPPED
+      reason: 'Chart is not signed : Signature verification not required'
+    - check: v1.0/chart-testing
+      type: Mandatory
+      outcome: FAIL
+      reason: |-
+        Chart Install failure: values don't meet the specifications of the schema(s) in the following chart(s):
+        postgresql-persistent:
+        enum items must be unique
+[snipped]
+    - check: v1.0/helm-lint
+      type: Mandatory
+      outcome: FAIL
+      reason: |
+        Helm lint has failed:  [INFO] Chart.yaml: icon is recommended
+        [ERROR] values.yaml: enum items must be unique
+        [ERROR] templates/: values don't meet the specifications of the schema(s) in the following chart(s):
+        postgresql-persistent:
+        enum items must be unique
+[snipped]
+    - check: v1.1/has-kubeversion
+      type: Mandatory
+      outcome: FAIL
+      reason: Kubernetes version is not specified
+    - check: v1.1/images-are-certified
+      type: Mandatory
+      outcome: FAIL
+      reason: |-
+        Failed to certify images : Failed to get images, error running helm template : values don't meet the specifications of the schema(s) in the following chart(s):
+        postgresql-persistent:
+        enum items must be unique
+[snipped]
+    - check: v1.0/contains-values
+      type: Mandatory
+      outcome: PASS
+      reason: Values file exist
+    - check: v1.0/not-contains-crds
+      type: Mandatory
+      outcome: PASS
+      reason: Chart does not contain CRDs
+```
+
+Address all issues detected by chart-verifier.
+
+## Create Pull Request
+
+When Helm chart is ready to merge generate and update [index.yaml](./index.yaml) and add them into Pull Request.
```

Comment:
This sentence doesn't make sense. I cannot understand it. Please use the commas `,` and `and` words more precisely so it's clear what you meant by that.

Reply:
I have removed the section at all.

Reply:
https://github.com/sclorg/helm-charts/pull/16/commits/cfc1f664a51ea2d6e78f9fe43d755f1f694a7126